packages/spec-parser/src/utils.ts

```diff
@@ -36,10 +38,11 @@ export class Utils {
     paramObject: OpenAPIV3.ParameterObject[],
     isCopilot: boolean
```

Comment:
the method is shared for api me, type b, custom copilot?

Reply:
Yes, in the future, will separate logic for different project type